packages/discordx/README.md

```diff
@@ -71,7 +71,7 @@ yarn add discordx
 
 # 💡 Why discordx?
 
-With `discordx`, we intend to provide latest upto date package to build bots with many features, such as multi-bot, simple commands, pagination, music etc. **Updated daily with discord.js changes**.
+With `discordx`, we intend to provide latest up to date package to build bots with many features, such as multi-bot, simple commands, pagination, music etc. **Updated daily with discord.js changes**.
```

Comment:
Should be "the  latest up to date", it's missing "the"?

Reply:
```suggestion
With `discordx`, we intend to provide the latest up-to-date package to easily build feature-rich bots with multi-bot compatibility, simple commands, pagination, music, and much more. **Updated daily with discord.js changes**.
```